src/pages/settings/Subscription/CardSection/BillingBanner/EarlyDiscountBanner.tsx

```diff
@@ -118,16 +118,18 @@ function EarlyDiscountBanner({isSubscriptionPage, onboardingHelpDropdownButton,
     }
 
     const title = isSubscriptionPage ? (
-        <Text style={styles.textStrong}>
-            {translate('subscription.billingBanner.earlyDiscount.subscriptionPageTitle.phrase1', {discountType: discountInfo?.discountType})}&nbsp;
-            <Text>{translate('subscription.billingBanner.earlyDiscount.subscriptionPageTitle.phrase2')}</Text>
-        </Text>
+        <RenderHTML
+            html={translate('subscription.billingBanner.earlyDiscount.subscriptionPageTitle', {
+                discountType: discountInfo?.discountType,
+            })}
+        />
     ) : (
         <View style={[styles.justifyContentBetween, styles.flexRow]}>
-            <Text style={(styles.textStrong, styles.flexShrink1)}>
-                {translate('subscription.billingBanner.earlyDiscount.onboardingChatTitle.phrase1')}&nbsp;
-                <Text>{translate('subscription.billingBanner.earlyDiscount.onboardingChatTitle.phrase2', {discountType: discountInfo?.discountType})}</Text>
-            </Text>
+            <RenderHTML
+                html={`<strong>${translate('subscription.billingBanner.earlyDiscount.onboardingChatTitle', {
```

Comment:
I don't mind having styling tags like `<strong />` outside of the translations file. I think it's even a good practice where it's practical (i.e: tags wrap the whole phrase). But it looks like you might've changed the styling here?